src/plugins/intel_gpu/src/plugin/ops/interpolate.cpp

```diff
@@ -64,14 +48,14 @@ static void CreateInterpolateOp(Program& p, const std::shared_ptr<ngraph::op::v4
         bool correctAxes =
             ((axes.size() == 2 || axes.size() == 4) && inputRank < 5) &&
             ((axes[0] == 0 && axes[1] == 1) ||
-             (axes[0] == 1 && axes[1] == 0) ||
-             (axes[0] == 2 && axes[1] == 3) ||
-             (axes[0] == 3 && axes[1] == 2));
+                (axes[0] == 1 && axes[1] == 0) ||
```

Comment:
Please keep original indentation if there is no problem.

Reply:
Fixed indentation